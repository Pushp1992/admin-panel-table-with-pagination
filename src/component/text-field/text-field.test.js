import {render, screen, cleanup} from '@testing-library/react';
import renderer from 'react-test-renderer';
import TextField from './text-field';
import '@testing-library/jest-dom';

afterEach(() => {
    cleanup()
});

it('match snapshot with DOM tree', () => {
    const component = renderer.create(<TextField/>);
    const DOMTree = component.toJSON();
    expect(DOMTree).toMatchSnapshot();
});

it('render TextField component', () => {
    render(<TextField/>);

    const textElement = screen.getAllByTestId('text-field');
    expect(textElement[0]).toBeInTheDocument();
});