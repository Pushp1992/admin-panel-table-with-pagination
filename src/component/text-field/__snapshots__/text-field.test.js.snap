// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`match snapshot with DOM tree 1`] = `
<input
  data-testid="text-field"
  type="text"
/>
`;
